gtsam/hybrid/HybridFactor.h

```diff
@@ -136,6 +136,10 @@ class GTSAM_EXPORT HybridFactor : public Factor {
   /// Return only the continuous keys for this factor.
   const KeyVector &continuousKeys() const { return continuousKeys_; }
 
+  /// Virtual class to compute tree of linear errors.
```

Comment:
linear?

Reply:
This linear because it only accepts `VectorValues` right? This is also why its definition in `HybridNonlinearFactor` throws a runtime error.